charts/rancher-istio/100.1.1+up1.11.7/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Istio
+  catalog.cattle.io/kube-version: 1.19 - 1.22
+  catalog.cattle.io/namespace: istio-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
```

Comment:
this annotations should be dropped since we're no longer auto-matching against rancher-kiali-server-crd, which was the one that was providing this GVR in the past. This seems to have been a change introduced in 1.11.7 that was not caught in previous rcs.

Reply:
e.g. this annotation got dropped https://github.com/rancher/charts/blob/632681382cae5ca209917d06967c19cd28ce9928/charts/rancher-istio/100.0.2%2Bup1.10.4/Chart.yaml#L2

Reply:
@aiyengar2 I was not aware of this annotation. I removed this annotation along with the crd-template directory. Could you please review it again?